apps/docs/content/components/content/Avatar.mdx

```diff
@@ -1,48 +1,117 @@
 ---
-title: Avatar (WIP)
-description: TBD
+title: Avatar
+description: Avatars are used to represent a user, team or another entity. They are often paired with text where room is available.
 category: "content"
 links:
-    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/TBD/src/TBD.tsx
+    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/Avatar/src/index.tsx
 ---
 
+<Example src="Avatar/docs/preview" isOpen />
+
 ## Props
 
-TODO: Add the props table, if we have multiple props table (ex: TagList and Tag) in the same file,  use the name of the component in ### before the each props table
+<PropTable component="Avatar" />
 
-## Guidelines
+<FeatureFlag flag="next">
+    ## Guidelines
 
-TODO: If we have some guidelines about this component's usage
+    TODO: If we have some guidelines about this component&apos;s usage
 
-### Accessibility ?
+    ### Accessibility ?
 
-TODO: If we have some guidelines about this component and accessibility
+    TODO: If we have some guidelines about this component and accessibility
+</FeatureFlag>
 
-## Anatomy
+<FeatureFlag flag="rc">
+    ## Anatomy
 
-TODO: We have anatomy screenshots from the Figma, we could most likely use them here
+    TODO: We have anatomy screenshots from the Figma, we could most likely use them here
 
-### Concepts
+    ### Concepts
 
-TODO: links to related concepts
+    TODO: links to related concepts
+</FeatureFlag>
 
-### Composed Components
+## Examples
 
-TODO: links to related components (most likely all component which are used by the slots)
+### Local Image
 
-## Examples
+It would normally be imported from a file like this: `import Frog from "./hadfield.png";` then used as the src.
+
+<Example src="Avatar/docs/localImage" />
+
+### Remote image
+
+An Avatar can fetch a remote image.
+
+<Example src="Avatar/docs/remoteImage" />
+
+### Fallback image
+
+A fallback image can be set in case the src fails to load.
+
+<Example src="Avatar/docs/fallbackImage" />
+
+### When src fails to load and fallback is not set
```

Comment:
I would make smaller titles as it'll be weird the sidebar. You could take inspiration from the example titles.